### ISSUE TITLE
Training set

### ISSUE THREAD

**Issue by small-winnie** [lartpang/HDFNet#6]
excuse me, I have a question about the training part, 2185 images as a training set and then tested on seven datasets, then 800 images of DUT trained as a training set and then tested on the test set of DUT

**Comment by lartpang** [comment]
@small-winnie 

At the time of this work, there were still no widely accepted training and testing settings in the RGBD SOD. So following some earlier work, we trained and tested independently on both parts of these datasets. That is, **the model trained on DUT-RGBD-TR was used for testing on DUT-RGBD-TE**, while **the model trained on NJUD+NLPR was used for evaluation on other datasets**.

**Comment by small-winnie** [comment]
Thank you for your answer.
